src/pages/SettingsPage.tsx

```diff
@@ -23,6 +44,13 @@ const SettingsPage = (props: RouteComponentProps) => {
         <div />
         <div />
       </ButtonBar>
+      <Modal isOpen={showResetBallotAlert}>
+        <Text>Are you sure you want to remove all votes and start over?</Text>
```

Comment:
change text from `remove all votes` to `clear all selections`

Reply:
fixed